dnf-behave-tests/dnf/search-sort.feature

```diff
@@ -9,7 +8,7 @@ Background:
 Scenario: sort alphanumerically
   When I execute dnf with args "search name"
   Then the exit code is 0
-   And stdout is
+   And dnf4 stdout is
```

Comment:
Do we need this dnf4/dnf5 distinction in the main branch? I've already started to remove it from tests that I newly enable for dnf5. I thing dnf4 is not supposed to be tested by this branch.